test/playwright.fixtures.ts

```diff
@@ -91,24 +91,22 @@ registerWorkerFixture('httpService', async ({}, test) => {
   ]);
 });
 
-const getExecutablePath = browserName => {
+registerWorkerFixture('defaultBrowserOptions', async ({browserName}, test) => {
+  let executablePath = undefined;
   if (browserName === 'chromium' && process.env.CRPATH)
-    return process.env.CRPATH;
+    executablePath = process.env.CRPATH;
   if (browserName === 'firefox' && process.env.FFPATH)
-    return process.env.FFPATH;
+    executablePath = process.env.FFPATH;
   if (browserName === 'webkit' && process.env.WKPATH)
-    return process.env.WKPATH;
-};
-
-registerWorkerFixture('defaultBrowserOptions', async ({browserName}, test) => {
-  const executablePath = getExecutablePath(browserName);
+    executablePath = process.env.WKPATH;
 
   if (executablePath)
     console.error(`Using executable at ${executablePath}`);
   await test({
     handleSIGINT: false,
     slowMo: options.SLOW_MO,
     headless: options.HEADLESS,
+    chromiumSandbox: Boolean(process.env.PW_CHROMIUM_SANDBOX),
```

Comment:
where's `PW_CHROMIUM_SANDBOX` set?

Reply:
Good catch, lets reverse it because it was set by accident to the right value (false).